Dockerfile

```diff
@@ -39,4 +39,6 @@ RUN \
     --wildcards '*/stack' && \
   rm stack.tgz && \
   mv stack /usr/local/bin/ && \
+  stack upgrade --git && \
```

Comment:
This is kind of dicey. Is there any way to upgrade to a specific SHA?

Reply:
Not that I can tell, but it's only done once and it's "alpha"